lib/elixir/lib/calendar/iso.ex

```diff
@@ -213,7 +213,7 @@ defmodule Calendar.ISO do
   end
 
   def date_to_iso_days(year, month, day) when year in -9999..9999 do
-    true = day <= days_in_month(year, month)
+    ensure_day_in_month(year, month, day)
```

Comment:
Should it be `ensure_day_in_month!` (with bang)? I'm not sure what's the convention when there is only the "raise" version.

Reply:
Yup, with bang!